versioned_docs/version-2.x/advanced/payments/authorize-net.mdx

```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 3
+title: Authorize.net
+description:
+  This guide explains how Front-Commerce allows using Authorize.net in a
+  headless commerce project.
+---
+
+import SinceVersion from "@site/src/components/SinceVersion";
+
+<p>{frontMatter.description}</p>
+
+Authorize.net is a payment gateway service that allows customers to securely
+make online payments. It provides a seamless integration for processing credit
+card transactions.
+
+## Magento1 (OpenMage LTS)
+
+<SinceVersion tag="2.26" />
+
+### Prerequisites
+
+This feature is directly provided by Magento (OpenMage LTS), but requires the
+Front-Commerce Magento module in version >=
+[1.5.0](https://gitlab.blackswift.cloud/front-commerce/magento1-module-front-commerce/-/releases/1.5.0).
+
+Update the Front-Commerce module if needed.
```

Comment:
**thought:** I feel like that's implicit enough already. I'm not sure it makes sense to add this line; if the version is a requirement, integrator should know they have to have the correct version.

**suggestion(non-blocking):** Remove this line.

Reply:
I removed the line :+1: